primitives/io/src/lib.rs

```diff
@@ -98,7 +98,7 @@ pub enum KillStorageResult {
 }
 
 /// Interface for accessing the storage from within the runtime.
-#[runtime_interface]
+#[runtime_interface(feature_force_version=old_state,root,1)]
```

Comment:
the usage of `feature_force_version` must be documented on the runtime_interface macro doc.